src/OidcClientSettings.ts

```diff
@@ -12,6 +12,7 @@ const DefaultClientAuthentication = "client_secret_post";
 const DefaultResponseMode = "query";
 const DefaultStaleStateAgeInSeconds = 60 * 15;
 const DefaultClockSkewInSeconds = 60 * 5;
+const DefaultOidcMetadataUrlPath = ".well-known/openid-configuration";
```

Comment:
This isn't a default value, it's part of the protocol. I think you could actually just get rid of this variable and inline it in the constructor below.